docs/toc.md

```diff
@@ -1,164 +1,93 @@
-# [Getting started](getting-started/index.md)
-## [What is the .NET Core Platform?](getting-started/what-is-dotnet.md)
-## [Installing .NET Core](getting-started/installing/index.md)
-### [.NET Core System Requirements](getting-started/installing/system-reqs.md)
-### [Installing .NET Core on Windows](getting-started/installing/installing-core-windows.md)
-### [Installing .NET Core on Linux](getting-started/installing/installing-core-linux.md)
-### [Installing .NET Core on OS X](getting-started/installing/installing-core-osx.md)
-## [Using the CLI tools to write console apps](getting-started/cli-console-app-tutorial.md)
-## [Setting up your integrated development environment](getting-started/devenv/index.md)
-### [🔧 Development using Visual Studio 2015](getting-started/devenv/using-visual-studio.md)
-### [🔧 Development using Visual Studio Code](getting-started/devenv/using-visual-studio-code.md)
-### [🔧 Development using vim](getting-started/devenv/using-vim.md)
-### [🔧 Development using Sublime Text](getting-started/devenv/using-sublime.md)
+# [About .NET](about/index.md)
+## [.NET Products](about/products.md)
+## [Community](about/community.md)
+## [Open Source](about/open-source.md)
+## [Learning Resources](about/learn.md)
+## [Talks](about/talks.md)
+# [Handbook](handbook.md)
```

Comment:
It seems counterintuitive to me to find "Fundamentals" AFTER "Handbook" and to find both "Handbook" and "Guide" in the same TOC.  It's not clear what difference in content you would expect in a "Handbook" vs. a "Guide", and "Fundamentals" implies something you should be looking at or familiar with first before you go on to other things.

I would replace "Handbook" here with ".NET Fundamentals" (including ".NET" in the title since we're introducing various .NET products in "About" so we do need to clarify what products subsequent TOC nodes are referring to).  Then the conceptual docs under Fundamentals can go in a Concepts section presented first, while the how-tos currently under Fundamentals can still be under Fundamentals following the Concepts section.

 

- Getting Started (became "About")
- Fundamentals
        -  Concepts (goes to primer)
         - First how-to topic or topic grouping
- Second how-to topic or topic grouping
- Scenarios (how to content for applications scenarios)
- C# Guide
- .NET Core Guide